.circleci/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2017-2021 City of Espoo
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eu
+
+cd "$( dirname "${BASH_SOURCE[0]}")"
+
+TEST_IMAGE="evaka/test:local"
+CIRCLECI_CLI_VERSION="0.1.16535"
+docker build -t "$TEST_IMAGE" - <<EOF
+FROM cimg/ruby:3.0.2
+RUN cd /tmp \
+ && curl -sSfL "https://github.com/CircleCI-Public/circleci-cli/releases/download/v${CIRCLECI_CLI_VERSION}/circleci-cli_${CIRCLECI_CLI_VERSION}_linux_amd64.tar.gz" -o circleci-cli.tar.gz \
+ && tar xzf circleci-cli.tar.gz \
+ && sudo mv "circleci-cli_${CIRCLECI_CLI_VERSION}_linux_amd64/circleci" /usr/local/bin \
+ && sudo chown 0:0 /usr/local/bin/circleci \
+ && rm -r circleci-cli.tar.gz "circleci-cli_${CIRCLECI_CLI_VERSION}_linux_amd64/"
+WORKDIR /ci
+EOF
+
+CHECK_COMMAND="erb -T- /ci/template.yml | tee /dev/tty | circleci --skip-update-check config validate -"
+
+if test -z "${1:-}" || [ "${1:-}" = "fail" ]; then
```

Comment:
What's the point of `${x:-}`? It seems to me that it just substitutes an empty string with an empty string 🤔 

If `${1:-}` is actually required, then this would be a simpler way:
```sh
arg=${1:-}
if [ -z "$arg" -o "$arg" = "fail" ]; then
```

Reply:
End result is the same, but maybe nicer with variable. I'll change it.

Point is that we want empty string - not undefined, because we use "u" in set -eu that will fail if we have "undefined" variable so "$1" would fail if argument is not provided.